06-files-input-and-output/remove_files.py

```diff
@@ -2,33 +2,43 @@
 
 import os
 
-def find_files_in_folder(path, extention, file_max_size):
+def find_files_in_folder(path, extension, file_max_size):
+    print(
+        "Searching for files with .{} extension in \"{}\" and maximum size of {}bytes".format(
+            extension, path, file_max_size
```

Comment:
Put each argument in a separate line and add `,` to the last argument as well. This is as defined in the style guide.